src/main/kotlin/racingcar/InputValidator.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+object InputValidator {
+	private const val MAX_NAME_LENGTH = 5
+	private const val MIN_TRY_COUNT = 1
+
+	fun validateName(names: List<String>) {
+		names.forEach() { name ->
+			when {
+				name.isEmpty() -> throw IllegalArgumentException("참가자 이름은 비어있을 수 없습니다.")
+				name.length > MAX_NAME_LENGTH -> throw IllegalArgumentException("참가자 이름은 ${MAX_NAME_LENGTH}자 이하여야 합니다.")
+			}
+		}
+	}
```

Comment:
반복문을 돌리지 않고 아래처럼 any를 활용해 더 코틀린스럽게 표현할 수도 있을 것 같아요!

```suggestion
    if (names.any { it.isEmpty() }) {
        throw IllegalArgumentException("참가자 이름은 비어있을 수 없습니다.")
    }
    if (names.any { it.length > MAX_NAME_LENGTH }) {
        throw IllegalArgumentException("참가자 이름은 ${MAX_NAME_LENGTH}자 이하여야 합니다.")
    }
```

Reply:
오 이 방법도 깔끔하고 좋은 것 같네요!! 조언 감사합니다!!